recipe/meta.yaml

```diff
@@ -0,0 +1,138 @@
+{% set name = "qtbase" %}
+{% set version = "6.7.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  - url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/{{ name }}-everywhere-src-{{ version }}.tar.xz
+    sha256: c5f22a5e10fb162895ded7de0963328e7307611c688487b5d152c9ee64767599
+    folder: {{ name }}
+    patches:
+      - patches/0001-Disable-unity-build-for-permission-plugins-in-macos.patch
+      - patches/0002-avoid-using-macos-12-resource.patch
+
+  - url: https://download.qt.io/development_releases/prebuilt/llvmpipe/windows/opengl32sw-64-mesa_12_0_rc2.7z  # [win]
+    sha256: 2a0d2f92c60e0962ef5f6039d3793424c6f39e49ba27ac04a5b21ca4ae012e15                                   # [win]
+    folder: opengl32sw                                                                                         # [win]
+
+build:
+  number: 0
+  skip: True  # [ppc64le or s390x]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+  ignore_run_exports:
+    - dbus    # [unix]
+  missing_dso_whitelist:
+    - '*/api-ms-win-shcore-scaling-l1-1-1.dll'     # [win]
+    - '*/api-ms-win-core-winrt-l1-1-0.dll'         # [win]
+    - '*/api-ms-win-core-winrt-string-l1-1-0.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('libdrm-devel') }}               # [linux]
+    - {{ cdt('libglvnd-glx') }}               # [linux and not x86_64]
+    - {{ cdt('libglvnd-egl') }}               # [linux and not x86_64]
+    - {{ cdt('libice-devel') }}               # [linux]
+    - {{ cdt('libsm-devel') }}                # [linux]
+    - {{ cdt('libx11-devel') }}               # [linux]
+    - {{ cdt('libxau-devel') }}               # [linux]
+    - {{ cdt('mesa-libgl-devel') }}           # [linux]
+    - {{ cdt('mesa-libgbm') }}                # [linux]
+    - {{ cdt('mesa-libegl-devel') }}          # [linux]
+    - {{ cdt('mesa-dri-drivers') }}           # [linux]
+    - {{ cdt('xcb-util-devel') }}             # [linux]
+    - {{ cdt('xcb-util-image-devel') }}       # [linux]
+    - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
+    - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
+    - {{ cdt('xcb-util-wm-devel') }}          # [linux]
+    - {{ cdt('xorg-x11-proto-devel') }}       # [linux]
+    - patch       # [unix]
+    - pkg-config  # [unix]
+    - bison       # [linux]
+    - flex        # [linux]
+    - gperf       # [linux]
+    - jom         # [win]
```

Comment:
Is it used as a build tool? I do not see jom or jom install commands. Or does it invoked implicitly?

Reply:
it is referenced in the cmake files for windows. I decided to just keep it instead of tentatively remove to see if it would still build, as it is just a build tool, and the windows boxes take forever to build, and time was limited.